### DISCUSSION TITLE
Search

### DISCUSSION THREAD

**Discussion by juan-dvd** [TimGeyssens/Umbraco-v8-Community-Website-StarterKit#24] [General]
Any preference about search options, to build a robust search:

Lucene 
Elastic Search
Azure Search

**Comment by juan-dvd** [comment]
Some links
Examine:

> https://our.umbraco.com/documentation/reference/searching/examine

Azure Search:

>  https://our.umbraco.com/packages/developer-tools/moriyama-azure-search/

>  https://www.sdkits.com/examinex-azure-search-for-umbraco
> https://github.com/thecogworks/Cogworks.AzureSearch
> https://www.wearecogworks.com/blog/how-to-simplify-azure-search-implementations/

Elastic Search: 

> https://our.umbraco.com/packages/website-utilities/novicellexamineelasticsearch/

....
Any other options or custom code? Have anyone check  Solr wrapper?

**Reply by TimGeyssens** [reply]
This one is pretty nice: https://our.umbraco.com/packages/website-utilities/full-text-search-8/ (uses examine/lucene)

But also keen to learn more about the above options :)
